assets/bpmn-js.css

```diff
@@ -2,6 +2,13 @@
   --bjs-font-family: Arial, sans-serif;
 }
 
+.djs-container {
+  --breadcrumbs-item-color: var(--color-blue-205-100-50);
+  --breadcrumbs-arrow-color: var(--color-black);
+  --drilldown-fill-color: var(--color-white);
+  --drilldown-background-color: var(--color-blue-205-100-50);
```

Comment:
as these are only bpmn-js releated, can we move them to `.bjs-container`?

Reply:
Not really, because the `.bjs-container` is a parent of the `.djs-container`

![image](https://user-images.githubusercontent.com/9433996/141286439-4692f186-6a10-4981-9dbe-d618ac355b67.png)

The full color scheme is defined in the `.djs-container`, therefore they wouldn't be available in the `.bjs-container`.

Reply:
An alternative would be to define the colors again in the `.bjs-container`, but I'm not sure whether this makes sense.

Reply:
Ah, makes sense :+1: